docs/contributors/index.md

```diff
@@ -0,0 +1,243 @@
+# Contributors
+
+This page is targeted at people who wish to contribute to Jupyter Scheduler directly,
+
+Developers wishing to extend or override Jupyter Scheduler should instead refer
+to our {doc}`developer's guide </developers/index>`.
+
+## Development install
+
+The below sequence of commands will install a development environment for
+Jupyter Scheduler locally. Before running this, you should ensure that NodeJS is
+installed locally. The `jlpm` command is JupyterLab's pinned version of
+[yarn](https://yarnpkg.com/) that is installed with JupyterLab. You may use
+`yarn` or `npm` in lieu of `jlpm` below.
+
+```bash
+# Clone the repo to your local environment
+git clone https://github.com/jupyter-server/jupyter-scheduler.git
+
+# Change dir to the cloned project
+cd jupyter-scheduler
+
+# Install the project in editable mode
+pip install -e .
+
+# Link your development version of the extension with JupyterLab
+jupyter labextension develop . --overwrite
+
+# Server extension must be manually installed in develop mode
+jupyter server extension enable jupyter_scheduler
+
+# Rebuild extension Typescript source after making changes
+jlpm build
+```
+
+You can watch the source directory and run JupyterLab at the same time in
+different terminals to watch for changes in the extension's source and
+automatically rebuild the extension.
+
+```bash
+# Watch the source directory in one terminal, automatically rebuilding when needed
+jlpm watch
+# Run JupyterLab in another terminal
+jupyter lab
+```
+
+With the watch command running, every saved change will immediately be built
+locally and available in your running JupyterLab. Refresh JupyterLab to load the
+change in your browser (you may need to wait several seconds for the extension
+to be rebuilt).
+
+By default, the `jlpm build` command generates the source maps for this
+extension to make it easier to debug using the browser dev tools. To also
+generate source maps for the JupyterLab core extensions, you can run the
+following command:
+
+```bash
+jupyter lab build --minimize=False
+```
+
+## Development uninstall
+
+```bash
+# Server extension must be manually disabled in develop mode
+jupyter server extension disable jupyter_scheduler
+pip uninstall jupyter_scheduler
+```
+
+In development mode, you will also need to remove the symlink created by
+`jupyter labextension develop` command. To find its location, you can run
+`jupyter labextension list` to figure out where the `labextensions` folder is
+located. Then you can remove the symlink named `jupyter-scheduler` within that
+folder.
```

Comment:
Should probably expand on this a bit. 

In development mode, you will also need to remove the symlink created by
`jupyter labextension develop` command. 
```bash
# Find the location where labextensions folder is located
jupyter labextension list
```
Example output:
```bash
/opt/anaconda3/envs/jupyter-scheduler/share/jupyter/labextensions
        @jupyterlab/scheduler v1.1.4 enabled OK
```

Then you can remove the symlink named `jupyter-scheduler` within that
folder.
```
# Remove the symlink 
rm /opt/anaconda3/envs/jupyter-scheduler/share/jupyter/labextensions/jupyter_scheduler
```

Reply:
Done. 👍